### PR TITLE
amam/add_award

### PR DESCRIPTION
**Pull request by mustofa-binary**
add global brand magazine 2019 award

### CODE REVIEW

File: src/templates/static/home.jsx [hunk 1]
```diff
@@ -350,6 +350,10 @@ const Home = () => {
                                 <img className='responsive' src={it.url_for('images/pages/home/awards/gbm-2018.svg')} alt='Global Brands Magazine 2018' />
                                 <span className='caption'>{it.L('Global Brands Magazine')}</span>
                             </div>
+                            <div className='award-icon'>
+                                <img className='responsive' src={it.url_for('images/pages/home/awards/gbm-2019.svg')} alt='Global Brands Magazine 2019' />
```
Comment (oskar-binary) [inline src/templates/static/home.jsx:354]:
```suggestion
                                <img className='responsive' src={it.url_for('images/pages/home/awards/gbm-2019.svg')} alt={it.L('Global Brands Magazine 2019')}/>
```

File: src/templates/static/why_us.jsx [hunk 2]
```diff
@@ -156,6 +156,10 @@ const WhyUs = () => (
                             <img className='responsive' src={it.url_for('images/pages/home/awards/gbm-2018.svg')} alt='Global Brands Magazine 2018' />
                             <span className='caption'>{it.L('Global Brands Magazine')}</span>
                         </div>
+                        <div className='award-icon gr-padding-20 align-self-center horizontal-margin'>
+                            <img className='responsive' src={it.url_for('images/pages/home/awards/gbm-2019.svg')} alt='Global Brands Magazine 2019' />
```
Comment (oskar-binary) [inline src/templates/static/why_us.jsx:160]: same here

File: src/templates/static/home.jsx (hunk 1 above)
Comment (mustofa-binary) [reply src/templates/static/home.jsx:354]: in binary static, no alt tag for image is being translated actually.

Comment (oskar-binary) [reply src/templates/static/home.jsx:354]: why not?

Comment (mustofa-binary) [reply src/templates/static/home.jsx:354]: im not sure why 😅 .

Comment (mustofa-binary) [reply src/templates/static/home.jsx:354]: probably we are removing translation workload here

Comment (oskar-binary) [reply src/templates/static/home.jsx:354]: I see well I think that comes at the expense of SEO and accessibility

Comment (mustofa-binary) [reply src/templates/static/home.jsx:354]: ok, lets create a card to add translation for all alt image in binary static

**Pull request merged**